resources/views/components/editable.blade.php

```diff
@@ -0,0 +1,39 @@
+{{-- components.editable
+
+Render an editable input field
+
+--}}
+
+<div
```

Comment:
I loved this inline editing feature, it would be great to have it mostly in the TableView